### ISSUE TITLE
show only java's qualified name if kotlin type differ from java type

### ISSUE THREAD

**Issue by robstoll** [robstoll/atrium#300]
*Platform* (jvm, js, android):
*Extension* (none, kotlin 1.3, jdk8): none

## Code related feature
```
expect(null as List<Int>).notToBeNull()
```
the output:
```
expect: null
◆ not to be: null but instance of List (kotlin.collections.List) -- Class: List (java.util.List)
  » is: empty
```
We would like to change it to:
```
expect: null
◆ not to be: null but instance of List (kotlin.collections.List) -- Class: java.util.List
  » is: empty
```
or in other words remove `List`

Following the things you need to do:

*lib*
- [x] adopt `DetailedObjectFormatter.format(kClass)` for jvm
- [x] run `./gr build`, most likely specs fail and need to be adapted
- [x] README.md should also change once you can build successfully. Make sure you only commit relevant changes, depending on your used OS it might change a lot (e.g. if you use Windows, it changes the paths).

## Your first contribution?
- Write a comment `I'll work on this` if you would like to take this issue over. 
  This way we get the chance to revise the description in case things have changed in the meantime, we might give you additional hints and we can assign the task to you, so that others do not start as well.
- See [Your first code contribution](https://github.com/robstoll/atrium/blob/master/.github/CONTRIBUTING.md#your-first-code-contribution) for guidelines.  
- Do not hesitate to ask questions here or to contact us via [Atrium's slack channel](https://kotlinlang.slack.com/team/U3DE1TXKP) if you need help
  ([Invite yourself](https://slack.kotlinlang.org/) in case you do not have an account yet).

**Comment by Hubisco** [comment]
Hi, I'll try to  work on this :)

**Comment by robstoll** [comment]
implemented with 45419b08 and 2552f691
